src/main.py

```diff
@@ -103,40 +107,40 @@ async def on_message(message: Message) -> None:
             if bust_index == 0:
                 await message.channel.send("We start from 1 around here.")
                 return
-            if bust_index > len(bust.current_channel_content):
+            if bust_index > len(bc.current_channel_content):
                 await message.channel.send("There aren't that many tracks.")
                 return
             skip_count = bust_index - 1
 
-        await bust.command_play(message, skip_count)
+        await bc.play(message, skip_count)
 
     elif message_text.startswith("!form"):
-        if not bust.current_channel_content or not bust.current_channel:
-            await message.channel.send("You need to use !list first, sugar.")
+        if not bc.current_channel_content or not bc.current_channel:
+            await message.channel.send("You need to use !list first.")
             return
 
         # Pull the google drive link to the form image from the message (if it exists)
         command_args = message.content.split()[1:]
         if command_args:
-            await voting.command_form(message, google_drive_image_link=command_args[0])
+            await voting.form(message, google_drive_image_link=command_args[0])
         else:
-            await voting.command_form(message)
+            await voting.form(message)
```

Comment:
I might give these methods slightly more informative names while we're changing the names anyhow. Such as:

* `generate_form`
* `skip_song`
* `send_song_list`

Since they're no longer directly tied to the command names.

Reply:
I changed form() and skip(). Not changed list since that is waiting to be factored out in #130 anyways.